src/coreComponents/constitutive/fluid/multifluid/compositional/CompositionalMultiphaseFluidPVTPackage.cpp

```diff
@@ -104,7 +108,16 @@ void CompositionalMultiphaseFluidPVTPackage::postProcessInput()
   checkInputSize( m_componentCriticalPressure, NC, viewKeyStruct::componentCriticalPressureString() );
   checkInputSize( m_componentCriticalTemperature, NC, viewKeyStruct::componentCriticalTemperatureString() );
   checkInputSize( m_componentAcentricFactor, NC, viewKeyStruct::componentAcentricFactorString() );
-  checkInputSize( m_equationsOfState, NP, viewKeyStruct::equationsOfStateString() );
+
+  if( m_constantPhaseViscosity.empty() )
+  {
+    m_constantPhaseViscosity.resize( NP );
+    for( integer ip = 0; ip < NP; ++ip )
+    {
+      m_constantPhaseViscosity[ip] = 0.001;      // Default value = 1 cP
```

Comment:
should it be an input parameter?

Reply:
Most definitely. I will expose this in the xml in a follow up PR.